### PR TITLE
fix: Use deprecated object keys

### PR DESCRIPTION
**Pull request by KadirBalku**
needs further testing before Merge can be resolved

### CODE REVIEW

File: packages/viur-vue-utils/bones/edit/default/textBone.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <template v-if="state.editor">
     <ckeditor
-      v-if="boneState.bonestructure['validHtml']"
+      v-if="boneState.bonestructure['valid_html']"
```
Comment (sveneberth) [inline packages/viur-vue-utils/bones/edit/default/textBone.vue:4]: If you use an old core this should still work IMO, so I would support both:
```suggestion
      v-if="boneState.bonestructure['valid_html'] ?? boneState.bonestructure['validHtml']"
```

**Review by sveneberth — CHANGES_REQUESTED** [review]

**Comment by sveneberth** [comment]
Hello @KadirBalku @akelch , can we fix this soon? Currently I don't have a wysiwyg editor in my project :( ...

**Comment by KadirBalku** [comment]
> Hello @KadirBalku @akelch , can we fix this soon? Currently I don't have a wysiwyg editor in my project :( ...

does this change need to aplied to the viur admin ?

**Comment by sveneberth** [comment]
> > Hello @KadirBalku @akelch , can we fix this soon? Currently I don't have a wysiwyg editor in my project :( ...
> 
> has this change to be aplied to the viur admin ?

yes

**Pull request merged**